mappings/net/minecraft/util/math/PositionImpl.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_6210 net/minecraft/util/math/Location
+CLASS net/minecraft/class_6210 net/minecraft/util/math/PositionImpl
```

Comment:
this class is not implementation detail
`Location` is also a sponge name so yes

Reply:
Go check upstream

Reply:
`PositionImpl` is a long concerning hard to understand name

Reply:
But we hv to stay consistent with upstream names, ask fabriccord why is this name like this

Reply:
no we do not have to stay consistent with bad names

Reply:
yes screenhandler is gonna be changed in an upcoming pr, i hv stuff on my system